### ISSUE TITLE
Feature request: Multiple sets of control buttons

### ISSUE THREAD

**Issue by lvmajor** [mgildea/Multi-Step-Form-Js#20]
Someone asked me if it could be possible to add two sets of buttons for controlling the form process (next/prev buttons)

After a quick look I wondered if it could be done simply by removing the ".first()" call [here](https://github.com/mgildea/Multi-Step-Form-Js/blob/master/src/multi-step-form.js#L81)

What do you think?

**Comment by lvmajor** [comment]
Just tested it and it works fine.

**Edit**: There seems to be a bug causing validation of next step when using the "submit/enter" key on keyboard. I will try to find where it comes from.

**Comment by mgildea** [comment]
so just for my own understanding...a form could have 2 "next" buttons that act identically (for instance one on the top of the form one on the bottom)?

**Comment by lvmajor** [comment]
Exactly, for cases when the form is long enough to have scrolling required in a single step.

**Comment by lvmajor** [comment]
Okay so the problem is obviously caused by the fact that my first attempt (removing the .first()) call when finding the navigation, did register both and hence there are two handlers created.

**Comment by lvmajor** [comment]
Okay found a workaround, by only using the first registered button when using the key '13' (enter). Don't know if that's optimal though... 

Do you think that would be useful/ a potential use case worthy of PR?

**Comment by godwindaniel101** [comment]
can i please get a sample on how to use this
